EIPS/eip-draft_transaction_representation.md

```diff
@@ -0,0 +1,67 @@
+---
+eip: 7023
+title: Automatic Transaction Representation
+description: Living guidelines for automatically generating a human-readable representation of a transaction and its corresponding EIP-712 signature.
+author: Gavin John (@Pandapip1)
+discussions-to: https://ethereum-magicians.org/t/automatic-transaction-representation/14256
+status: Living
```

Comment:
```suggestion
status: Draft
```
All EIPs/ERCs should go through the same process, especially for EIPs authored by an EIP editor to avoid any perception of special treatment.